src/account_id_ref.rs

```diff
@@ -56,7 +56,7 @@ impl AccountIdRef {
     /// It is the responsibility of the caller to ensure the account ID is valid.
     ///
     /// For more information, read: <https://docs.near.org/docs/concepts/account#account-id-rules>
-    pub fn new_unchecked<S: AsRef<str> + ?Sized>(id: &S) -> &Self {
+    pub fn new_unvalidated<S: AsRef<str> + ?Sized>(id: &S) -> &Self {
```

Comment:
Let's make this method `pub(crate)`.

Let's remove the example from the `AccountIdRef` doc-string.

The safety note currently refers to the non-existing `new` method, please fix it; it is better to refer to [std::path::Path](https://doc.rust-lang.org/stable/std/path/struct.Path.html) implementation:

```
// Safety: see `AccountId::new`
```

Reply:
Not sure what this means, how is [this](https://github.com/ruseinov/near-account-id/blob/ca4ce096734c28ae524e6b8df12dfed4e825ff26/src/account_id_ref.rs#L41) non-existant?

Reply:
The `pub(crate)` change has been implemented.

Reply:
> Not sure what [this](https://github.com/ruseinov/near-account-id/blob/ca4ce096734c28ae524e6b8df12dfed4e825ff26/src/account_id_ref.rs#L41) means, how is this non-existent?

Ah, well, it should state:

```
// Safety: see `AccountIdRef::new`
```

not

```
// Safety: see `AccountId::new`
```

> The pub(crate) change has been implemented.

Will you push? 😄